src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionProvider.cs

```diff
@@ -139,5 +186,39 @@ public bool CheckRequestAcceptsCompression(HttpContext context)
             }
             return !string.IsNullOrEmpty(context.Request.Headers[HeaderNames.AcceptEncoding]);
         }
+
+        private readonly struct ProviderCandidate : IEquatable<ProviderCandidate>
+        {
+            public ProviderCandidate(string encodingName, double quality, int order, ICompressionProvider provider)
+            {
+                EncodingName = encodingName;
+                Quality = quality;
+                Order = order;
+                Provider = provider;
+            }
+
+            public string EncodingName { get; }
+
+            public double Quality { get; }
+
+            public int Order { get; }
```

Comment:
Priority?

Reply:
LOL, I had Priority and changed it to Order last minute 😂